src/languages/en.js

```diff
@@ -1074,7 +1074,7 @@ export default {
         publicDescription: 'Anyone can find this room',
         createRoom: 'Create room',
         roomAlreadyExistsError: 'A room with this name already exists',
-        roomNameReservedError: 'A room on this workspace already uses this name',
+        roomNameReservedError: 'Room names #admins and #announce are reserved names for this workspace',
         roomNameInvalidError: 'Room names can only include lowercase letters, numbers and hyphens',
```

Comment:
I followed your suggestion on reverting to two separate errors for restricted and existing room names and changed the roomNameReservedError message a little bit - I didn’t use the same pattern as in name’s containsReservedWord error message, because here we are checking for the exact #admins and #announce names and we are able to create a name including both words like e.g. #ck-admins. (Or maybe we should change the validation here?) If it doesn’t sound ok or you have an idea for a different message, please let me know, I’ll change it.

Reply:
Thanks, looks pretty good. From what I read in Slack I think from we want an error message for each default room name. We can use a translation function like this to achieve that. https://github.com/Expensify/App/blob/bb8dff159a7424000ff981e8301d5e63d2aa5064/src/languages/en.js#L64

`#admins is a default room on all workspaces. Please choose another name.`
`#announce is a default room on all workspaces. Please choose another name.`